docs/source/release/v5.0.0/indirect_geometry.rst

```diff
@@ -13,7 +13,7 @@ New
 ###
 
 - Simultaneous fitting has been added as an option on the convolutional, F(Q) and Msd fitting tabs in QENS Data Analysis. This allows multiple spectra from one or more reductions
-  to be fitted with parameters tied together. For more information on these changes, see the updated documentation :ref:`Indirect Data Analysis <interface-indirect-data-analysis>`.
+  to be fitted with parameters tied together. For more information on these changes, see the updated documentation :ref:`Inelastic Data Analysis <interface-inelastic-data-analysis>`.
```

Comment:
I wonder if this change to legacy docs will break them, as in this version won't the linked document still have the same name?

Reply:
If I don't change the reference, Jenkins build raised a warning leading to a build failure!

Reply:
Ah ok, I guess it links to the current version of the docs, not the 5.0 version of them

Reply:
Yes, it seems to be the case.

Reply:
Yes this is the slight awkwardness of having the release notes here too. The docs are a snapshot of a given point in time and as the code changes the old release notes become less valid at describing the current state. We generally just accept that this will happen and online we have access to the old versioned notes so it's possible to access the information.